onnx/defs/reduction/defs.cc

```diff
@@ -33,7 +34,10 @@ tensor has the same rank as the input if keepdims equal 1. If keepdims equal 0,
 the resulted tensor have the reduced dimension pruned.
 
 The above behavior is similar to numpy, with the exception that numpy default keepdims to
-False instead of True.)DOC";
+False instead of True.
+
+The attribute complement_axis is used in conjunction with the axis attribute, and has a default value of False.
```

Comment:
I suggest rewording this a little bit : 

The above behavior is similar to numpy, with the following exceptions
1. numpy defaults keepdims to False instead of True
2. This op uses complement _axes attribute which defaults to false but when set to true indicates all axes except the specified axes will be reduced.
The attribute complement_axis is used in conjunction with the axis attribute, and has a default value of False.